pkg/controller/directvolumemigration/task.go

```diff
@@ -174,6 +179,30 @@ func (t *Task) Run() error {
 		if err = t.next(); err != nil {
 			return liberr.Wrap(err)
 		}
+	case CleanStaleRsyncResources:
+		// TODO Need to add some labels during DVM run to differentiate
+		// deletion of rsync resources that are active vs stale. Using
+		// one label for both is the wrong approach.
+		err := t.deleteRsyncResources()
+		if err != nil {
+			return liberr.Wrap(err)
+		}
+		t.Requeue = NoReQ
+		if err = t.next(); err != nil {
+			return liberr.Wrap(err)
+		}
+	case WaitForStaleRsyncResourcesTerminated:
```

Comment:
@dymurray can we merge both cases in one case? looks like both are doing exactly same thing. 
```suggestion
	case WaitForStaleRsyncResourcesTerminated, WaitForRsyncResourcesTerminated:
```

Reply:
Yes... duh. Thanks